tests/ExcelMapper/Mappings/Readers/ColumnNameMatchingValueReaderTests.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace ExcelMapper.Mappings.Readers.Tests
+{
+    using System;
+
+    using Xunit;
+
+    public class ColumnNameMatchingValueReaderTests
+    {
+        [Fact]
+        public void Ctor_ColumnName()
+        {
+            var reader = new ColumnNameMatchingValueReader(e => e == "ColumnName");
+            Assert.NotNull(reader);
+        }
+
+        [Fact]
+        public void Ctor_NullColumnName_ThrowsArgumentNullException()
+        {
+            Assert.Throws<ArgumentNullException>("predicate", () => new ColumnNameMatchingValueReader(null));
+        }
+    }
+}
```

Comment:
Please add newline

Reply:
Hmm, I tried to add a new line but it seems to be removed when I commit it. Any idea?